build/isc-bind9/local.mog

```diff
@@ -0,0 +1,54 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+# Mediated binaries
+<transform file path=$(PREFIX)/(s?bin)/(.*) -> emit \
+    link path=$(OPREFIX)/%<1>/%<2> target=../$(PROG)-$(VERSION)/%<1>/%<2> \
+    mediator=$(PROG) mediator-version=$(VERSION) >
+
+# Mediated man pages
+<transform file path=$(PREFIX)/share/man/(man\d)/(.*) -> emit \
+    link path=$(OPREFIX)/share/man/%<1>/%<2> \
+    target=../../../$(PROG)-$(VERSION)/share/man/%<1>/%<2> \
+    mediator=$(PROG) mediator-version=$(VERSION) >
+
+# Restart fmri service
+<transform file path=$(PREFIX)/sbin/ \
+    -> set restart_fmri svc:/ooce/network/dns:named$(sVERSION)>
+
+# Set permissions for directories
+<transform file path=etc/$(PREFIX)/ -> set preserve renamenew>
```

Comment:
This needs to be moved down below the additional files below
(the order of entries here is important)

Reply:
I resolved this with @hadfl , and `renamenew` is working as it should be